runtime/doc/treesitter.txt

```diff
@@ -1285,7 +1285,7 @@ lint({buf}, {opts})                              *vim.treesitter.query.lint()*
     Use |treesitter-parsers| in runtimepath to check the query file in {buf}
     for errors:
     • verify that used nodes are valid identifiers in the grammar.
-    • verify that predicates and directives are valid.
+    • verify that guards and directives are valid.
```

Comment:
No, I think that refers (or should refer) specifically to `:h treesitter-predicates`, in parallel to `treesitter-directives`. I don't think "guards" is helpful here.